sirepo/template/srw.py

```diff
@@ -41,13 +41,15 @@
 
 _OUTPUT_FOR_MODEL = PKDict(
     coherenceXAnimation=PKDict(
+        check_backup=True,
```

Comment:
This will require a db upgrade. I think it is better if we deal with this programmatically inside srw.py, that is, in the code below (which already makes assumptions about the report name anyway) it adds code based on the report name.